tests/event-api-tests/observations/observations.spec.ts

```
@@ -0,0 +1,133 @@
+  const responseInfo = await executor
+    .sendRequest({ url, request, timeout: 10000 })
+    .then((res) => res.getResponseInfo());
+
+  expect(
+    responseInfo.status,
+    `Expect response status to be 200 for search request`
+  ).toBe(200);
+  const firstEvent = responseInfo.json?.data?.[0];
+  const firstObservationId = firstEvent?.observations?.[0];
+  expect(firstObservationId, "ObservationId should be defined").toBeDefined();
+  return firstObservationId!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)